entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd /var/www
+
+go get github.com/nitrous-io/goop
+goop install
+
```

Comment:
ideally the env would be plopped in when the container runs

so our app needs to accept env vars from the env